ppb/sprites.py

```diff
@@ -182,6 +250,11 @@ def bottom(self, value: Union[float, int]):
 
     @property
     def top_left(self) -> Vector:
+        """
+        The top left corner of the object.
+
+        Can be set to a :class:`ppb_vector.Vector`.
```

Comment:
Is this where we want to point this?

Reply:
It's good enough for now and we can come back and change it if we change it.